code/drasil-build/lib/Build/Drasil/Make/Print.hs

```diff
@@ -21,12 +21,21 @@ build (M rules) = addCommonFeatures rules $
 
 -- | Renders specific makefile rules. Called by 'build'.
 printRule :: Rule -> Doc
-printRule (R t d _ c) = printTarget t d $+$ printCmds c
+printRule (R c t d _ cmd) = printComments c $+$ printTarget t d $+$ printCmds cmd
+
+-- | Renders a makefile comment
+printComment :: MkComment -> Doc
+printComment [] = empty
+printComment c  = text ("# " ++ c ++ "\n")
```

Comment:
I can add it to this PR itself. I will make the changes now! If it's ok, I used the `Data.Text` and `Text.Wrap` libraries because I had some trouble carrying out the wrap text functions when using the `pretty` library.

Reply:
That's pretty neat! Nice work! That library looks like it will be useful in general for us with code generation. #3221 might be able to use it too.

Reply:
Thanks! Yeah, I can try to use this library in #3221 when I go to solve it.